uui-core/src/data/processing/views/Tree.ts

```diff
@@ -533,6 +533,17 @@ export class Tree<TItem, TId> {
         }
     }
 
+    public getParents(id: TId) {
+        const parentIds = this.getParentIdsRecursive(id);
+        const parents = parentIds.reduce<TItem[]>((acc, parentId) => {
+            if (!this.byId.has(parentId)) {
+                return acc;
+            }
+            return [...acc, this.byId.get(parentId)];
```

Comment:
The [...acc, something] clones the array. While this may be not significant in this case - there's not many parents, in general such pattern for .reduce() is considered as bad.

Reply:
Ok, that makes sense. I just knew that it is a reduced amount of parents here. Will change.